### PR TITLE
Track all routes in Piwik

### PR DESCRIPTION
**Pull request by Rikuoja**

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -8,13 +8,25 @@ import Home from 'views/Home';
 import Info from 'views/Info';
 import NotFound from 'views/NotFound';
 
+function trackLink() {
+  /* global Piwik */
+  try {
+    const tracker = Piwik.getTracker("http://analytics.hel.ninja/piwik/piwik.php", 2);
```
Comment (akx) [inline src/routes.js]: The tracking URL should be configurable. After all, this is supposed to be a reusable app...

Comment (Rikuoja) [inline src/routes.js]: K, what would be the best location for the tracking URL? All the way in kk.config and then get it to window via props?

Comment (akx) [inline src/routes.js]: Yeah. Maybe an `uiConfig` object in `kk.config` that gets plonked into the HTML, and is then read by the frontend code?

**Comment by akx** [comment]
So yeah, `getOptions` in `getSettings()` should check that if `_.isString(uiConfig)`, it's `JSON.parse(uiConfig)`'d at that point. That'll retain compatibility with envvar-based configuration.

File: src/config.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const config = {
   languages: ['fi'],
-  apiBaseUrl: (typeof window !== 'undefined' ? window.API_BASE_URL : null) || 'http://localhost:8000/'
+  apiBaseUrl: (typeof window !== 'undefined' ? window.API_BASE_URL : null) || 'http://localhost:8000/',
+  uiConfig: (typeof window !== 'undefined' ? window.UI_CONFIG : null) || null
```
Comment (akx) [inline src/config.js]: This does not need the ` || null` fallback. Also, add the trailing comma.

File: src/config.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 const config = {
   languages: ['fi'],
   apiBaseUrl: (typeof window !== 'undefined' ? window.API_BASE_URL : null) || 'http://localhost:8000/',
-  uiConfig: (typeof window !== 'undefined' ? window.UI_CONFIG : null) || null
+  uiConfig: typeof window !== 'undefined' ? window.UI_CONFIG : null,
```
Comment (Rikuoja) [inline src/config.js]: My IDE is vehemently against unneeded commas.

**Pull request merged**